tests/install_upgrade_operators/product_upgrade/conftest.py

```
@@ -107,31 +125,26 @@ def required_konflux_mirrors(cnv_target_version, cnv_current_version):
     machine_config_pools_conditions,
 ):
-    """Ensures the Konflux IDMS contains the required mirror entries for the CNV upgrade target version."""
+    """Ensures Konflux IDMS mirrors are set up if the IIB was built by Konflux pipeline."""
     if is_disconnected_cluster:
         LOGGER.warning("Skip applying IDMS in a disconnected setup.")
         return
 
-    if cnv_source == HOTFIX_STR:
-        LOGGER.info("IDMS updates skipped as upgrading using production source/upgrade to hotfix")
+    if iib_build_info.get("pipeline") != KONFLUX_PIPELINE:
+        LOGGER.warning(f"Pipeline is '{iib_build_info.get('pipeline')}', not Konflux. Skipping IDMS.")
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)